DOCS-1196: Modular resources review (#2116)
viamrobotics/docs: docs/modular-resources/_index.md

- Various types of hardware {{< glossary_tooltip term_id="component" text="components" >}}.
- High-level functionality exposed as {{< glossary_tooltip term_id="service" text="services" >}}.

> If the [component](/components/) or [service](/services/) you want to use for your project is not natively supported, you can [use _modular resources_ from the Viam Registry](#the-viam-registry) or [create your own modular resources](#create-your-own-modular-resources).

Author: Done

1. Package your modular resource or modular resources as a {{< glossary_tooltip term_id="module" text="module" >}} and [upload the module to the Viam registry](/modular-resources/upload/) to make it available for deployment to robots.
   You can upload _private_ modules for your [organization](/manage/fleet/organizations/) or _public_ modules.
1. Once you have uploaded your module to the registry, you can [deploy and configure the module](/modular-resources/configure/) from [the Viam app](https://app.viam.com/).

>    If the added resource is a component or service, you can then test it using the [Control tab](/manage/fleet/#remote-control) and [program](/program/) it with Viam's Go or Python SDKs.

Author: @andf-viam I was assuming that you'll now be able to program and test modular services like components, if I'm assuming incorrectly and this is just a quirk I'm unaware of LMK

Reviewer: Ah. I think the control tab only works for some components and services. But maybe that's not important to call out. How about
```suggestion
   You can test your added resource using the [Control tab](/manage/fleet/#remote-control) and [program](/program/) it with Viam's Go or Python SDKs.
```
